package Pro;
message QueryGateStateLG2GT{}